index.js

```diff
@@ -94,6 +94,16 @@ module.exports = () => {
 			return peekedMessages;
 		};
 
+		const peekActive = async (subscriptionId, n) => {
```

Comment:
I would just name it `peek`, since it is the original name in the API. It is intrinsic you peek active messages if not specified otherwise